CMakeLists.txt

```diff
@@ -22,6 +22,15 @@ include(CommonCppFlags)
 file(GLOB EASTL_SOURCES "source/*.cpp")
 add_library(EASTL ${EASTL_SOURCES})
 
+if (MSVC)
+	set(EASTL_NATVIS_DIR "doc")
```

Comment:
Could you fix this indentation?

Reply:
I've normalized the indentation to use 4 spaces for all new additions, but it looks like indentation in existing files is also inconsistent. Let me know it needs changing.

Reply:
Good point, we should probably fix that indentation at some point, but that's separate from this PR. Thanks for fixing it up for the added lines.